### PR TITLE
Create weweb 

### PR DESCRIPTION
**Pull request by EWMAs**
## What kind of change does this PR introduce?

Create new user doc article on how to integrate a Supabase backend in a WeWeb frontend.

## What is the current behavior?

Not applicable

## What is the new behavior?

Not applicable

## Additional context

This is the doc to include WeWeb in the Supabase Partner Program

### CODE REVIEW

File: apps/reference/docs/guides/integrations/weweb
Comment (awalias) [inline apps/reference/docs/guides/integrations/weweb:38]: might just wanna change these ones to be relative but otherwise looks great 👍

**Review by awalias — APPROVED** [review]

File: apps/reference/docs/guides/integrations/weweb
Comment (thorwebdev) [inline apps/reference/docs/guides/integrations/weweb:1]:
```suggestion
---
id: weweb
title: 'WeWeb'
description: 'Build user interfaces on top of existing databases.'
---

This guide explains how to connect a Supabase back-end to a WeWeb front-end and then configure all the CRUD operations necessary to build an Admin Portal with user authentication, roles, and permissions.
```

**Review by thorwebdev — CHANGES_REQUESTED** [review]
Hey, would you be able to rename the file to `weweb.mdx`, add the necessary metadata (see example ) and add it to the reference: https://github.com/supabase/supabase/blob/master/apps/reference/nav/_referenceSidebars.js#L261

**Comment by saltcod** [comment]
Hey @EWMAs we've just migrated our docs from Docusaurus to NextJS. Apologies for the inconvenience, but would you mind porting these changes to `/apps/docs` instead of `docs/reference`?  You could wait until you have approval here and then move them over, or just do it now would work as well.

Any questions please let me know

Thanks and apologies in advance!

**Comment by dannykng** [comment]
Closing in favor of https://github.com/supabase/supabase/pull/10217

**Pull request closed without merging**
